Row/col based access functions have been deprecated

```
/[...]/Arduino/hardware/keyboardio/avr/libraries/Kaleidoscope-LEDEffect-DigitalRain/src/Kaleidoscope-LEDEffect-DigitalRain.cpp: In member function 'virtual void kaleidoscope::plugin::LEDDigitalRainEffect::update()':
/[...]/Arduino/hardware/keyboardio/avr/libraries/Kaleidoscope-LEDEffect-DigitalRain/src/Kaleidoscope-LEDEffect-DigitalRain.cpp:29:74: warning: 'static void kaleidoscope::plugin::LEDControl::setCrgbAt(byte, byte, cRGB)' is deprecated:
------------------------------------------------------------------------
Row/col based access functions have been deprecated. Please use the KeyAddr/KeyAddr based versions instead.
------------------------------------------------------------------------
 [-Wdeprecated-declarations]
     ::LEDControl.setCrgbAt(row, col, getColorFromIntensity(map[col][row]));
                                                                          ^
In file included from /[...]/Arduino/hardware/keyboardio/avr/libraries/Kaleidoscope/src/Kaleidoscope-LEDControl.h:20:0,
                 from /[...]/Arduino/hardware/keyboardio/avr/libraries/Kaleidoscope-LEDEffect-DigitalRain/src/Kaleidoscope-LEDEffect-DigitalRain.h:3,
                 from /[...]/Arduino/hardware/keyboardio/avr/libraries/Kaleidoscope-LEDEffect-DigitalRain/src/Kaleidoscope-LEDEffect-DigitalRain.cpp:1:
/[...]/Arduino/hardware/keyboardio/avr/libraries/Kaleidoscope/src/kaleidoscope/plugin/LEDControl.h:90:40: note: declared here
   DEPRECATED(ROW_COL_FUNC) static void setCrgbAt(byte row, byte col, cRGB color) {
                                        ^
```

--- a/src/Kaleidoscope-LEDEffect-DigitalRain.cpp
+++ b/src/Kaleidoscope-LEDEffect-DigitalRain.cpp
@@ -26,7 +26,7 @@ namespace kaleidoscope { namespace plugin {
 				}
 
 				// Set the colour for this pixel
-				::LEDControl.setCrgbAt(row, col, getColorFromIntensity(map[col][row]));
+				::LEDControl.setCrgbAt(KeyAddr(row, col), getColorFromIntensity(map[col][row]));
 			}
 		}
 

